hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -97,13 +97,13 @@
                    the FIFOs, the CDC's, the core state machine and the shift register.
                    In the current implementation, the CDC FIFOs are drained not reset.
                    Therefore software must confirm that both FIFO's empty before releasing
-                   clearing this bit.''',
+                   the IP from reset.''',
```

Comment:
Could we add a description of how to reset the IP in the programmer's guide?

It sounds like I'd have to set SW_RST, then wait for the FIFOs to drain(?), then clear SW_RST?